### DISCUSSION TITLE
superset 2.1.0 frontend start to use the "npm run dev-server", will appear many warnings

### DISCUSSION THREAD

**Discussion by gufenqing** [apache/superset#27235] [General]
npm run dev-server will appear many warnings 
![npmrundevservererror](https://github.com/apache/superset/assets/77658522/2a707aa1-3411-4f5a-bcad-c04c29eba57f)

![image](https://github.com/apache/superset/assets/77658522/0510fb1e-e00b-4dc7-9af4-f42add32ec01)

I hope that will get the assistance to how to solves the warning and correctly see the superset page

**Comment by michael-s-molina** [comment]
Hi @gufenqing. Superset 2.x versions are not supported anymore. For that reason, I'm moving this to a discussion. I highly recommend upgrading your Superset version.
